tfidf_text_executor.py

```diff
@@ -28,9 +43,39 @@ def __init__(
             )
 
     @requests
-    def encode(self,docs: DocumentArray,  *args, **kwargs) -> DocumentArray:
-        iterable_of_texts = docs.get_attributes('text')
-        embedding_matrix = self.tfidf_vectorizer.transform(iterable_of_texts)
+    def encode(self, docs: Optional[DocumentArray],  *args, **kwargs):
```

Comment:
what about `parameters` to override `traversal_path`

Reply:
good catch, updated